build.gradle

```diff
@@ -1,14 +1,3 @@
-apply plugin: 'kotlin'
-apply plugin: 'java'
-apply plugin: 'maven'
-apply plugin: 'idea'
-
-group = 'se.zensum'
-version = '1.0-SNAPSHOT'
-description = 'Receive incoming HTTP requests/webhooks and write them to a Kafka topic'
-
-defaultTasks 'run'
-
 buildscript {
```

Comment:
Do you still need shadow jar, when we using jib?

Reply:
Yes, until we can figure out how to use JIB with Docker-compose